### ISSUE TITLE
Support for multiline commands

### ISSUE THREAD

**Issue by zhogov** [wulfgarpro/history-sync#4]
Currently tool breaks multiline commands. Example:
Before `zhsync`:
```
: 1475279111:0;vi
: 1475279824:0;mkdir tmprepo\
cd tmprepo\
touch file.txt
: 1475279852:0;ls
```
After `zhsync` will become
```
: 1475279111:0;vi
: 1475279824:0;mkdir tmprepo\
: 1475279852:0;ls
cd tmprepo\
touch file.txt
```
See - command 1475279824 is broken

Fix: replace `sort -u` with something that will respect multiline commands:
https://github.com/wulfgarpro/history-sync/blob/master/history-sync.plugin.zsh#L56

Not bash specialist, but I think something like [this](http://stackoverflow.com/questions/11315889/how-to-sort-with-multiple-lines-in-bash) can help

**Comment by wulfgarpro** [comment]
Yes, you're correct; sort doesn't play nicely with history entries that have multi-line delimiters.  Patch incoming.

**Comment by wulfgarpro** [comment]
[0001-added-awk-line-to-retain-multi-line-history-in-histo.patch.txt](https://github.com/wulfgarpro/history-sync/files/852091/0001-added-awk-line-to-retain-multi-line-history-in-histo.patch.txt)

**Comment by wulfgarpro** [comment]
Fixed in a44eb956829996f638018246efcc55976c32f3c7.

**Comment by mullnerz** [comment]
Still not working perfectly, see  #7
